### PR TITLE
Test

### PR DESCRIPTION
**Pull request by cpsbottestuser**
@tranandh87 @priya006 

This is test PR

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -13,5 +13,5 @@
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.web.container"/>
 	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.module.container"/>
-	<classpathentry kind="output" path="build/classes"/>
+	<classpathentry kind="output" path="build/classe"/>
```
Comment (tranandh87) [inline .classpath:16]: comment 1

Comment (cpsbottestuser) [inline .classpath:16]: Done

Comment (tranandh87) [inline .classpath:16]: comment 2

Comment (cpsbottestuser) [inline .classpath:16]: Done

**Pull request closed without merging**